modifytags/modifytags_test.go

```diff
@@ -0,0 +1,133 @@
+package modifytags
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+var update = flag.Bool("update", false, "update golden (.out) files")
+
+// This is the directory where our test fixtures are.
+const fixtureDir = "./test-fixtures"
+
+func TestApply(t *testing.T) {
+	var tests = []struct {
+		file  string
+		m     *Modification
+		start token.Pos
+		end   token.Pos
+	}{
+		{
+			file: "all_structs",
+			m: &Modification{
+				Add:       []string{"json"},
+				Transform: SnakeCase,
+			},
+			start: token.Pos(1),
```

Comment:
These can be set to token.NoPos as well, defaulting to node.Start()

And since token.NoPos is 0, you can just omit the field when it doesn't need to be set.